extensions/arc/src/ui/dashboards/miaa/miaaDashboardOverviewPage.ts

```diff
@@ -345,4 +329,24 @@ export class MiaaDashboardOverviewPage extends DashboardPage {
 			!this._miaaModel.configLastUpdated &&
 			!this._miaaModel.databasesLastUpdated;
 	}
+
+	private refreshDashboardLinks(): void {
+		const kibanaEndpoint = this._controllerModel.getEndpoint(Endpoints.logsui);
+		if (kibanaEndpoint && this._miaaModel.config) {
+			const kibanaQuery = `kubernetes_namespace:"${this._miaaModel.config.metadata.namespace}" and custom_resource_name :"${this._miaaModel.config.metadata.name}"`;
+			const kibanaUrl = `${kibanaEndpoint.endpoint}/app/kibana#/discover?_a=(query:(language:kuery,query:'${kibanaQuery}'))`;
+			this._kibanaLink.label = kibanaUrl;
+			this._kibanaLink.url = kibanaUrl;
+			this._kibanaLoading!.loading = false;
+		}
+
+		const grafanaEndpoint = this._controllerModel.getEndpoint(Endpoints.metricsui);
+		if (grafanaEndpoint && this._miaaModel.config) {
+			const grafanaQuery = `var-hostname=${this._miaaModel.info.name}-0`;
+			const grafanaUrl = grafanaEndpoint ? `${grafanaEndpoint.endpoint}/d/40q72HnGk/sql-managed-instance-metrics?${grafanaQuery}` : '';
```

Comment:
@bergeron I had to update the MIAA dashboard link here too - I imagine it's likely the same for Postgres. So just note that for when you fix up the PG dashboard